Src/Coravel/Scheduling/Schedule/Interfaces/IScheduledEventConfiguration.cs

```diff
@@ -68,6 +71,11 @@ public interface IScheduledEventConfiguration
         /// <returns></returns>
         IScheduledEventConfiguration PreventOverlapping(string uniqueIdentifier);
 
+        /// <summary>
+        /// Restrict task to run only if result of <paramref name="predicate"/> is true.
```

Comment:
"Restrict scheduled task to run only when result of <paramref name="predicate"/> is true."

Just being really nit-picky because this will be exposed via intellisence to anyone using Coravel 👍